src/components/messenger/list/styles.scss

```diff
@@ -324,9 +324,29 @@ $side-padding: 16px;
     color: theme.$color-greyscale-12;
   }
 
+  &__button-container {
+    display: flex;
+    margin: 24px 0px;
+  }
+
   &__create {
     width: 100%;
-    margin: 16px 0px;
+
+    div {
+      display: flex;
+      align-items: center;
+      gap: 8px;
+
+      &__button-icon {
+        color: theme.$color-secondary-11;
+      }
+    }
+
+    &:hover {
+      &__button-icon {
+        color: theme.$color-black;
+      }
+    }
```

Comment:
Is this one needed? Seems like the hover color should already come from the Button component.

Reply:
@dalefukami, correct, with the hot-fix you made to zUI this is no longer necessary. I have removed the `button-icon` class and the styles here. As you said, the colour and hover colour comes from the Button component and works as expected now with the latest zUI version. Thank you!